mathesar/state/django.py

```diff
@@ -53,9 +52,9 @@ def reflect_db_objects(metadata, db_name=None):
             models.Schema.current_objects.filter(database=database).delete()
 
 
-def sync_databases_status():
+def sync_databases_status(databases):
```

Comment:
Before we were reflecting the status of every database, irrespective of whether or not we're intending to reflect only a single database. Now we'll only check the status of the database we've singled out.